src/style.css

```diff
@@ -0,0 +1,184 @@
+.calendar {
+  box-sizing: border-box;
+  padding: 10px;
+  width: 732px;
```

Comment:
The width of the calendar is set to a fixed value of 732px. This might not be responsive on smaller screens. Consider using relative units like percentages or media queries to make the layout more flexible.